#! /bin/sh /usr/share/dpatch/dpatch-run
## 83-skip-orbsvcs-examples-and-tests.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Prevent make from building orbsvcs tests, examples and performance-tests.
## DP: This should have been done by the regeneration of the GNumakefile, but 
## DP: seemingly a bug in mwc.pl prevented this from working. Therefore, we 
## DP: manually remove these directories.

@DPATCH@

Index: ACE_wrappers/TAO/orbsvcs/GNUmakefile
===================================================================
--- ACE_wrappers.orig/TAO/orbsvcs/GNUmakefile	1970-07-23 03:47:18.000000000 +0100
+++ ACE_wrappers/TAO/orbsvcs/GNUmakefile	2004-10-31 16:02:53.409266744 +0100
@@ -42,7 +42,4 @@
 	$(KEEP_GOING)@cd Dump_Schedule && $(MAKE) -f $(MAKEFILE) $(@)
 	$(KEEP_GOING)@cd CosEvent_Service && $(MAKE) -f $(MAKEFILE) $(@)
 	$(KEEP_GOING)@cd Concurrency_Service && $(MAKE) -f $(MAKEFILE) $(@)
-	$(KEEP_GOING)@cd performance-tests && $(MAKE) -f $(MAKEFILE) $(@)
-	$(KEEP_GOING)@cd tests && $(MAKE) -f $(MAKEFILE) $(@)
-	$(KEEP_GOING)@cd examples && $(MAKE) -f $(MAKEFILE) $(@)
 
